huxley/core/admin/assignment.py

```diff
@@ -41,29 +42,44 @@ def load(self, request):
         reader = csv.reader(assignments['csv'])
 
         def get_model(model, name, cache):
-            if not name in cache:
-                cache[name] = model.objects.get(name=name)
+            try:
+                if not name in cache:
+                    cache[name] = model.objects.get(name=name)
+            except ObjectDoesNotExist:
```

Comment:
This is very minor, since we're only executing one fetch in the `try` block, but each model has its own `DoesNotExist` exception type. I used it in the example above. I wouldn't comment ordinarily, but while you're making the above change, sneak this in there :)